src/main/java/com/example/petbuddybackend/service/care/state/CareStateMachine.java

```diff
@@ -6,22 +6,29 @@
 import com.example.petbuddybackend.repository.care.CareRepository;
 import com.example.petbuddybackend.utils.exception.throweable.general.StateTransitionException;
 import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
+import java.time.Duration;
+import java.time.LocalDate;
 import java.util.List;
 
 @Service
 @RequiredArgsConstructor
 public class CareStateMachine {
 
-    private static final String ACCEPTED_CARE_EDIT_MESSAGE = "Cannot edit care that has been accepted by caretaker";
+    private static final List<CareStatus> statusesThatCanBeCancelledBySystem =
+            List.of(CareStatus.PENDING, CareStatus.ACCEPTED);
 
-    private static final List<CareStatus> globalStatusesThatCanBeCancelled =
-            List.of(CareStatus.PENDING, CareStatus.ACCEPTED, CareStatus.AWAITING_PAYMENT);
+    private static final List<CareStatus> statusesThatCanBeOutdatedBySystem =
+            List.of(CareStatus.PENDING, CareStatus.ACCEPTED);
```

Comment:
to tutaj ready to proceed tez powinno byc

Reply:
bo jezeli jest ready to proceed a start_date minal i opiekun nie zaznaczyl jako completed to nie ustawimy tego jako outdated

Reply:
Aa w sumie tak